### PR TITLE
CORE-5586

### PR DESCRIPTION
**Pull request by cincuranet**

### CODE REVIEW

File: src/msgs/history2.sql [hunk 1]
```diff
@@ -1144,7 +1144,7 @@ message to appear.', 'Your program encountered an input or output error.', 'c_pg
 (1149, 'hvlad', '2006-02-02 19:15:11', 17, 44, 'Current memory = !c
 Delta memory = !d
 Max memory = !x
-Elapsed time= !e sec', NULL, NULL, 'c_pg');
+Elapsed time = !e sec', NULL, NULL, 'c_pg');
 (1150, 'hvlad', '2006-02-03 19:17:10', 17, 45, 'Cpu = !u sec
```
Comment (hvlad) [inline src/msgs/history2.sql]: No need to change history records :)

Comment (cincuranet) [inline src/msgs/history2.sql]: I saw it there, so I played safe card. Should I remove this change?

Comment (romansimakov) [inline src/msgs/history2.sql]: @hvlad Is there a reason to keep them?

Comment (hvlad) [inline src/msgs/history2.sql]: history2.sql contains old versions of changed messages. Formally, you should put pre-edited version of message as new record there, and never change existing records. But i really not sure it (history2.sql) is used somehow these days...

Comment (dyemanov) [inline src/msgs/history2.sql]: I'd vote for deleting the history file. We have VCS logs anyway.

Comment (asfernandes) [inline src/msgs/history2.sql]: I'd go further and delete more things not useful, as the fields for file and function name in messages2.sql. We have grep and others tools anyway.

Comment (hvlad) [inline src/msgs/history2.sql]: No objection fro my side, but this is offtop here ;)

Comment (cincuranet) [inline src/msgs/history2.sql]: So. Should I revert the `history2.sql` for this PR or could it stay as it's going to be deleted anyway?

Comment (dyemanov) [inline src/msgs/history2.sql]: Revert, plz.

Comment (cincuranet) [inline src/msgs/history2.sql]: Done.

**Pull request merged**